lbcapi/api.py

```diff
@@ -44,9 +51,14 @@ def call(self, method, url, params={}, stream=False, files=None):
         if url.startswith(self.server):
             url = url[len(self.server):]
 
-        # Convert parameters into list of tuples.
-        # This makes request encoding to keep the order
-        params = params.items()
+        if sys.version_info <= (3, 0):
+            # If not py3 - convert parameters into list of tuples.
+            # This makes request encoding to keep the order.
+            # In py3 "requests" lib fails with list of tuples,
+            # so the dict is necessary.
```

Comment:
Nice catch, but I wonder if this causes problems with HMAC and GET combination. The URL encoding of parameters is done in two separate locations: First for signature calculation and then for request. If there is no strict order of parameters, then in theory the order of URL encoding might differ, and that would result to failing signature. I believe it is very unlikely, but it would be nice to be 100 % sure.

Maybe the request could be created first, just like in case of POST...